v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/spannerchangestreamstobigquery/SpannerChangeStreamsToBigQuery.java

```diff
@@ -202,6 +203,14 @@ private static void validateOptions(SpannerChangeStreamsToBigQueryOptions option
     if (options.getDlqRetryMinutes() <= 0) {
       throw new IllegalArgumentException("dlqRetryMinutes must be positive.");
     }
+    if (Objects.equals(
```

Comment:
Can this just be `options.getBigQueryChangelogTableNameTemplate().equals(BigQueryUtils.BQ_CHANGELOG_FIELD_NAME_TABLE_NAME)`

Reply:
done.